core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -416,15 +416,14 @@ export class BaseUser {
   }
 
   /**
-   * This function validates whether an anchor tag is correctly linked
-   * to external PDFs or not. Use this particularly when interacting with
-   * buttons associated with external PDF links, because Puppeteer,
-   * in headless-mode, does not natively support the opening of external PDFs.
+   * Validates whether an anchor tag correctly links to external PDFs or LinkedIn profiles.
```

Comment:
I think this function to be used for links that cannot be opened directly other than just pdf and LinkedIn page. I suggest changing it to this:

```
This function validates whether an anchor tag correctly links to external PDFs or links that cannot be opened directly. 
Puppeteer, in headless mode, does not natively support opening external PDFs.
```